posthog/migrations/0803_move_recording_annotation_to_comments.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 4.2.22 on 2025-07-23 22:32
+
+from django.db import migrations
+
+# Bulk update every CHUNK_SIZE items
+CHUNK_SIZE = 500
+
+
+# NB: sort of duplicates migration 789
+# we were still creating saved filters on team creation with no type set
+# so we end up with rows in DB with no type
+# uses bigger chunks than 789
+# we know for this migration that all the rows with no type
+# are definitely saved filters and not collections
+def migrate_recording_annotations(apps, schema_editor):
+    Annotation = apps.get_model("posthog", "Annotation")
+    Comment = apps.get_model("posthog", "Comment")
+
+    recording_annotations = Annotation.objects.filter(scope="recording", deleted=False)
+
+    comment_chunk = []
+    annotation_chunk = []
+    for annotation in recording_annotations.iterator(chunk_size=CHUNK_SIZE):
+        comment_chunk.append(
+            Comment(
+                content=annotation.content,
+                scope="recording",
+                item_id=annotation.recording_id,
+                item_context={
+                    "is_emoji": annotation.is_emoji,
+                    "time_in_recording": annotation.date_marker.isoformat(),
+                    "migrated_from_annotation": annotation.id,
```

Comment:
keep track of which annotation it came from in case we need to unpick a bug in the migration